service/src/main/java/uk/nhs/adaptors/gpc/consumer/web/RequestBuilderService.java

```diff
@@ -29,6 +29,12 @@ public SslContext buildSSLContext() {
             .buildSSLContext();
     }
 
+    @SneakyThrows
```

Comment:
Consider creating a subclass SdsRequestBuilderService extends RequestBuilderService that overrides buildSSLContext() to achieve the desired behaviour. Just adding extra methods creates violates single-responsibility and creates a sort of interface bloat. What if we now need 20 different services each with their own request builder logic?

Reply:
For this one we no longer need the other method, so I have just removed